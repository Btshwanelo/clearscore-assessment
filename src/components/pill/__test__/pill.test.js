import React from "react";
import Enzyme from "enzyme";
import { shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

import Pill from "../index";

Enzyme.configure({ adapter: new Adapter() });

describe("pill component", () => {
  it("should render correctly", () => {
    const component = shallow(
      <Pill track_status={""}>
        {" "}
        <>...</>
      </Pill>
    );

    expect(component.debug()).toMatchSnapshot();
  });
});
