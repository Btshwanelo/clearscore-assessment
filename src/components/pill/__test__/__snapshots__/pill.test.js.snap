// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pill component should render correctly 1`] = `"<ForwardRef(Typography) variant=\\"p\\" component=\\"div\\" color=\\"primary.clr_orange_pill_text\\" backgroundColor=\\"custom.orange_pill_background\\" sx={{...}} />"`;
